### PR TITLE
feat(delivery_report): Do not genererate delivery report for analysis…

### PR DESCRIPTION
**Pull request by henrikstranneheim**
This PR adds/fixes:

- Skips generating a delivery report for analyses with a started_at date prior to 2017-09-27 (when statusDB was initilized on Hasta)

### How to prepare for test
- [x] ssh to hasta

### How to test
- [x] login to hasta
- [x] do `us`
1. `cg upload delivery-reports -p`
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh skip_delivery_report_prior_to_hasta_2`
1. `cg upload delivery-reports -p`

### Expected test outcome
1.  check that more cases than betterbison and freeelf are included in the delivery report generation ✅ 
1. check taht only betterbison and freeelf are included in the delivery report generation ✅ 
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [x] tests executed by HS
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
### How to test
- [x] login to hasta
- [x] do `us`
1. `cg upload delivery-reports -p`
Screenshot 2020-09-29 at 15 11 29

- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh skip_delivery_report_prior_to_hasta_2`
1. `cg upload delivery-reports -p`
Screenshot 2020-09-29 at 15 08 21

File: tests/store/api/status/test_analyses_to_delivery_report.py [hunk 1]
```diff
@@ -42,3 +43,25 @@ def test_outdated(analysis_store, helpers):
...
     assert analysis in analyses
+
+
+def test_outdated_analysis(analysis_store, helpers):
+    """Tests that analyses that are older then when Hasta became production (2017-09-26) are not included in the cases to generate a delivery report for"""
+
+    # GIVEN an analysis that is older than Hasta
+    timestamp_old_analysis = get_date("2017-09-26")
+    timestamp = datetime.now()
+    delivery_timestamp = timestamp - timedelta(days=1)
```
Comment (moonso) [inline tests/store/api/status/test_analyses_to_delivery_report.py:54]: Could we add some more GIVEN about the initial state?

**Review by moonso — APPROVED** [review]
👍 Nice if we can get rid of the cronjobs, well done!

**Comment by henrikstranneheim** [comment]
Deployed on hasta:
Screenshot 2020-09-30 at 13 25 33

**Comment by emmser** [comment]
Nice!

**Comment by patrikgrenfeldt** [comment]
Hasta has purchase date 2018-06-01 according to it's logbook

**Pull request merged**